src/test/java/crystal/task/DeadlineTest.java

```diff
@@ -0,0 +1,23 @@
+package crystal.task;
+
+import crystal.CrystalException;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Represents the DeadlineTest class.
+ *
+ */
+public class DeadlineTest {

```

Comment:
Maybe try using Test names  according to convention `featureUnderTest_testScenario_expectedBehavior()`